apps-rendering/src/renderer.ts

```diff
@@ -242,7 +242,12 @@ const textElement =
 		);
 		switch (node.nodeName) {
 			case 'P': {
+				if (text === '* * *') {
```

Comment:
I wonder if we should extract this to a helper? it's also appearing in:

* [`components/HeadingTwo`](https://github.com/guardian/dotcom-rendering/blob/main/apps-rendering/src/components/HeadingTwo/index.tsx#L131)
* [`renderer`](https://github.com/guardian/dotcom-rendering/blob/main/apps-rendering/src/renderer.ts#L130)

Reply:
Yeah I think you're right, it's a bit scattered.

In the heading code we're actually already doing this correctly, it's not rendering the `h2` if there's a dinkus.

Reply:
I've merged this fix for now, but we should consider the linked issue as a better long-term solution.

Reply:
I think it’s also possible to get `***` or `•••`, according to this check in DCR:

https://github.com/guardian/dotcom-rendering/blob/93597a1d00613f641d4415e19946418ae2ed513c/dotcom-rendering/src/model/enhance-dividers.ts#L15-L20